PythonControl.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Are we passing `Simulated Trajectory` back to the `Wheel Controller` from `Robot Simulator`? Wouldn't we just be passing the current state?

Reply:
What was intended here is to show that the performance metric are used to change the wheel controller. I've updated the diagram to reflect this. The diagram is more to show how the controller is designed and updated the C firmware than the operation of the controller.